facebook-messenger.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pry'
```

Comment:
@anam-mansoor7 @taniasoaressantos 
Not related at all to this piece of code, but I didn't change the current version of the gem. Mainly because I am not sure what to pursue here. Should we come up with our own versioning? Or leave the current version as a remnant of the version we forked from the original?